src/Board.java

```diff
@@ -0,0 +1,65 @@
+public class Board {
+
+    private char[] spaces = {'1', '2' ,'3' ,'4' ,'5' ,'6' ,'7' ,'8' ,'9'};
+
+
+    public Board(){
+        this.spaces = spaces;
+    }
+
+    public void playerOneMove(int space){
+        spaces[space - 1] = 'x';
+    }
+
+    public void playerTwoMove(int space){
+        spaces[space - 1] = 'o';
+    }
+
+    public void computerMove(){
+        for(int i = 0; i < spaces.length; i++){
+            if(i != 'x' && i != 'o'){

```

Comment:
believe this should be
```suggestion
            if(spaces[i] != 'x' && spaces[i] != 'o'){

```